index.test.js

```diff
@@ -0,0 +1,74 @@
+const http = require('http');
+const supertest = require('supertest');
+const handler = require('./src/handler');
+const parseEnv = require('./src/parse-env');
+
+const describeIfEnv = (process.env.AIRTABLE_API_KEY && process.env.AIRTABLE_BASE_ID) ? describe : xdescribe;
+
+const getExistingPosts = async () => {
+  const config = parseEnv(process.env);
+  const server = http.createServer(handler(config));
+
+  return (await supertest(server).get('/v0/Posts')).body;
+};
+
+describeIfEnv('integration', () => {
+  this.config = parseEnv(process.env);
+
+  describe('allowedMethods "*"', () => {
+
+    beforeEach(() => {
+      this.config.allowedMethods = '*';
+      this.server = http.createServer(handler(this.config));
+    });
+
+    it('Can make GET request', async () => {
+      this.response = await supertest(this.server)
+        .get('/v0/Posts');
+
+      expect(this.response.statusCode).toBe(200);
+      expect(this.response.body).toBeTruthy();
```

Comment:
I opted not to test the whole body content or to manually reset the table each time.

On one hand, testing every attribute would be more rigorous, but on the other it would make it more brittle and I'm not sure how much value it would add at this point.

If we later implement a feature like #10 then maybe we should make the tests more thorough.

As for resetting the table every time, it would just take a lot longer. If you think it's necessary, I'm not strongly opposed though.

Reply:
Starting by testing for a valid response makes sense to me. It still gives us a lot of value really easily.

We can explore resetting the testing Airtable base or using snapshot tests to validate responses in a later PR. This PR can stay focused on the testing structure.